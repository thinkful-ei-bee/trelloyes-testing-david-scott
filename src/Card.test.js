import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card.js';
import renderer from 'react-test-renderer';


describe('<Card />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title="x" content="z"/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('matches snapshot', () => {
    const tree =renderer.create(<Card title='x' content='z'/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
