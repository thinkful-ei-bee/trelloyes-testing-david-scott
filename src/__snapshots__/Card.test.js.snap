// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> matches snapshot 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    x
  </h3>
  <p>
    z
  </p>
</div>
`;
